### ISSUE TITLE
Dynamic inventories refreshed only once an hour

### ISSUE THREAD

**Issue by msperl** [rundeck-plugins/ansible-plugin#237]
A rescan of the inventory only happens once an hour - or so it seems.

Is there an option to:
* change the interval
* trigger an immediate reload of the inventory (a new host shows up)

**Comment by fdevans** [comment]
Sorry we missed this. If it's still a problem please open a new Issue with current Rundeck version details, and a deeper description of how this can be repeated.
